### PR TITLE
etcd-local-disk: move it to day 2

### PR DESCRIPTION
**Pull request by EmilienM**
Move the operation on day 2 as it was really unstable on day 1.
Therefore we remove some unneeded bits in `ipi-install-install`.

### CODE REVIEW

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.15-e2e-openstack-ccpmso-zone

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.15-e2e-openstack-ccpmso-zone

**Review by vrutkovs — APPROVED** [review]
/approve
/lgtm

**Comment by EmilienM** [comment]
/hold
I know Martin wanted to have a look too.

File: ci-operator/step-registry/openstack/provision/etcd-on-local-disk/openstack-provision-etcd-on-local-disk-commands.sh [hunk 1]
```diff
@@ -4,21 +4,64 @@ set -o nounset
...
     exit 0
 fi
 
 if [[ "${USE_RAMFS}" == "true" ]]; then
-    echo "INFO: USE_RAMFS is set to true, skipping etcd on local disk configuration"
+    info "INFO: USE_RAMFS is set to true, skipping etcd on local disk configuration"
     exit 0
 fi
 
+if ! openstack flavor show "${OPENSTACK_CONTROLPLANE_FLAVOR}" -f value -c "OS-FLV-EXT-DATA:ephemeral" | grep -qE '^(10|[1-9]\d+)$'; then
```
Comment (EmilienM) [inline ci-operator/step-registry/openstack/provision/etcd-on-local-disk/openstack-provision-etcd-on-local-disk-commands.sh:60]: we should also check the alternate flavor used by CCPMSO e2e.

**Comment by mandre** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.15-e2e-openstack-ccpmso-zone

**Review by mandre — COMMENTED** [review]
/lgtm
Just waiting for the job to come back to remove the hold.

**Comment by EmilienM** [comment]
I don't think `e2e-openstack-ccpmso-zone` is going to pass until https://github.com/openshift/cluster-control-plane-machine-set-operator/pull/263 has merged.

/pj-rehearse

**Pull request merged**